sdk/typescript/src/providers/void-provider.ts

```diff
@@ -63,8 +63,8 @@ export class VoidProvider extends Provider {
     throw this.newError('getReferenceGasPrice');
   }
 
-  async getSuiSystemState(): Promise<SuiSystemState> {
-    throw this.newError('getSuiSystemState');
+  async getCurrentEpochStaticInfo(): Promise<SuiSystemState> {
```

Comment:
nit: can we also rename the return type?

Reply:
We will actually change the return type completely. That would be done in a separate PR because I will have to refactor a whole lot of things